### DISCUSSION TITLE
Enable console with recent update

### DISCUSSION THREAD

**Discussion by tomasbargut** [mbround18/valheim-docker#307] [Help!] [answered]
Hi everyone!

With the new update, the console is disable by default and only can be enabled with the option "-command"
There is a way to pass this option to the script that start the server?

**Comment by mbround18** [comment, marked as answer]
Is this a new thing from valheim devs? ruddee why they no put that in the update notes ;-;

**Reply by mbround18** [reply]
@tomasbargut can you toss a link to where this is documented? 0.o

**Reply by tomasbargut** [reply]
I see it in the valheim discord.

**Reply by mbround18** [reply]
Whats it do ?

**Reply by mbround18** [reply]
* Ingame console disabled by default (add launch argument "-console" to enable)


That update is for client only and not server

**Reply by mbround18** [reply]
Thats purely for desktop game only, it has no effect on the dedicated server.

**Reply by mbround18** [reply]
LOL i asked about it and love the confidence in the their community XP 
![image](https://user-images.githubusercontent.com/12646562/112347473-1f156680-8c84-11eb-82e4-7e60189eb137.png)

**Reply by tomasbargut** [reply]
Ok thanks for the answer! 

There must be some other reason why I can't open the console in my instance

**Reply by Finomnis** [reply]
@tomasbargut I had the same problem.
I found out those things are necessary to access the console:

- specify the '-console' launch argument to your *client*. Not the server.
- Enter yourself in the admin list of the server

It then worked for me. Without the '-console' command in the client it didn't.
